internal/daemon/controller/handlers/managed_groups/managed_group_service.go

```diff
@@ -601,12 +601,13 @@ func validateCreateRequest(req *pbs.CreateManagedGroupRequest) error {
 			attrs := req.GetItem().GetOidcManagedGroupAttributes()
 			if attrs == nil {
 				badFields[globals.AttributesField] = "Attribute fields is required."
-			}
-			if attrs.Filter == "" {
-				badFields[attrFilterField] = "This field is required."
 			} else {
-				if _, err := bexpr.CreateEvaluator(attrs.Filter); err != nil {
-					badFields[attrFilterField] = fmt.Sprintf("Error evaluating submitted filter expression: %v.", err)
+				if attrs.Filter == "" {
```

Comment:
Could also make use of the `GetFilter()` method that should protect against `nil`